core/src/main/scala/org/apache/spark/BarrierTaskContext.scala

```diff
@@ -85,28 +84,28 @@ class BarrierTaskContext private[spark] (
         // BarrierCoordinator on timeout, instead of RPCTimeoutException from the RPC framework.
         timeout = new RpcTimeout(365.days, "barrierTimeout"))
 
-      // messages which consist of all barrier tasks' messages
-      var messages: Array[String] = null
       // Wait the RPC future to be completed, but every 1 second it will jump out waiting
       // and check whether current spark task is killed. If killed, then throw
       // a `TaskKilledException`, otherwise continue wait RPC until it completes.
-      try {
-        while (!abortableRpcFuture.toFuture.isCompleted) {
+
+      // import scala Success locally to avoid conflict with org.apache.spark.Success
+      import scala.util.{Failure, Success, Try}
```

Comment:
nit: you can rename `s.u.Success` to `ScalaSuccess`, then you can move this import declaration to the beginning.

Reply:
+1

Reply:
sure.